packages/bpk-component-accordion/src/bpk-accordion-item.scss

```diff
@@ -28,21 +28,22 @@
     padding: $bpk-spacing-xs 0;
     border: 0;
     background-color: transparent;
-    color: $bpk-link-color;
     text-align: left;
     cursor: pointer;
     appearance: none;
 
+    @include bpk-themeable-property(color, --bpk-accordion-color, $bpk-link-color);
```

Comment:
This is interesting ... part of me thinks that these should be covered by `themeAttributes` from `bpk-component-link`.

Reply:
but the end interface is so much better and consistent if the provide the accordion colour IMHO

Reply:
From a design system perspective though, if we want to theme links, we should theme all links.

Reply:
Not sure where to draw the line here, if accordion is using a link under the hood should be theme explicitly as part of the accordion and not having some external factor (e.g. change the theme for link) theming part of this component

Shall I change it?

Reply:
As discussed offline, lets just leave as is.